apps/openchallenges/challenge-service/src/main/resources/db/migration/V1.0.0__create_tables.sql

```diff
@@ -53,6 +51,20 @@ CREATE TABLE `challenge`
 -- KEY                `FKk9w2ogq595jbe8r2due7vv3xr` (`account_id`),
 -- CONSTRAINT `FKk9w2ogq595jbe8r2due7vv3xr` FOREIGN KEY (`account_id`) REFERENCES `banking_core_account` (`id`)
 
+-- challenge_organization_role definition
+
+CREATE TABLE `challenge_organization_role`
```

Comment:
What do you think about renaming this table `challenge_contribution_role` to make it more flexible. This would open the door to non-organization contribution, e.g. org that participated to a challenge (future feature?).